### DISCUSSION TITLE
Feed loading failed

### DISCUSSION THREAD

**Discussion by Cansiny0320** [community/community#67274] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Feed loading failed,I have tried many times

https://github.com/conduit/for_you_feed  return 502 status code

![image](https://github.com/community/community/assets/51797219/7170b96a-5ee1-426d-875d-97a44c344fa3)

![image](https://github.com/community/community/assets/51797219/7160e102-97b0-4e12-9099-358eda56f045)

**Comment by Cansiny0320** [comment]
X-Github-Request-Id:
F75E:2A27:345277F:397FAAD:650561F4

**Comment by TodePond** [comment]
To be honest this feels like an improvement

**Comment by ashishkumarpalai** [comment]
i have same issues

**Comment by mybearworld** [comment]
beautiful

**Comment by sryze** [comment]
This is a sign from above that we just should go back to the old feed.

**Comment by mybearworld** [comment]
Sad, it works again

**Comment by ramin-guliyev** [comment]
Same issue

**Comment by hridaya2004** [comment]
I am facing the same issue.

**Comment by Cansiny0320** [comment]
It's normal now, it looks like this bug has been fixed.
